tests/unit/TechRecordsService.unitTest.ts

```diff
@@ -430,7 +440,7 @@ describe("insertTechRecord", () => {
         delete techRecord.trailerId;
         delete techRecord.techRecord[0].statusCode;
 
-        await techRecordsService.setTrailerId(techRecord);
+        techRecord.trailerId = await techRecordsService.setTrailerId();
```

Comment:
why not just test the output of the function directly?
expect(await techRecordsService.setTrailerId()).toEqual("C530001");

Reply:
done